src/crystal/event_loop/file_descriptor.cr

```diff
@@ -7,7 +7,7 @@ abstract class Crystal::EventLoop
     # `nil`.
     #
     # Returns the system file descriptor or handle, or a system error.
```

Comment:
```suggestion
    # Returns the system file descriptor or handle and the actual
    # blocking state, or a system error.
```

Reply:
We must return the blocking value because:

- it may not be the same value (event-loop can decide otherwise);
- win32 needs to save the actual value as `@system_blocking` in the `File` object;